### PR TITLE
Contact number security updates

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

**Review by Nimphal — APPROVED** [review]

File: server/helpers/utilities.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function getAuthErrorDescription(error) {
     }`,
   )
   let type =
-    'Service temporarily unavailable. Please try again later. If this issue persists, please contact the Helpdesk on 0203 788 4636.'
```
Comment (andrewrlee) [inline server/helpers/utilities.js:32]: Guess you could potentially look at centralising the phone number and put it into config?

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "applicationinsights": "^1.8.7",
     "applicationinsights-native-metrics": "0.0.5",
     "audit-ci": "^3.1.1",
-    "axios": "^0.19.0",
+    "axios": "^0.21.1",
```
Comment (andrewrlee) [inline package.json:33]: there is a possible breaking changed that was introduced around this where a parameter is no longer supported for posts. Have slacked you regarding this

**Review by andrewrlee — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (andrewrlee) [reply package.json:33]: Looks like just using GETS and specifying the whole url so should be fine

**Pull request merged**